Software_Carbon_Intensity/Appendix_A.md

```diff
@@ -1 +1,101 @@
+
+
+2) Hosting Infrastructure - Dedicated Vs Shared
+
+Monitoring techniques will also vary based on the hosting mode for the software. For those with dedicated infrastructure, SCI will just be the sum total of Operational Emission values across the different layers. In the equation for SCI ( SCI per unit of baseline = (E*I) + M , the value of M does not make an impact when calculating delta carbon intensity = Current CI- Original CI since the hardware is exclusively reserved for the purpose of the said software. Hence the monitoring technique can potentially look for variances in the Operational Emissions value and variances of it to raise alert for the Site Reliability Engineer.
+
+The situation is different when we consider Shared infrastructure servers, multi-tenant databases, SaaS software shared by multiple customers. Here multiple micro-services could share the same PaaS compute platforms and storage services which by design is carbon friendly. In these cases, the percentage of allocation of infrastructure  is necessary information to be able to calculate the carbon intensity value for the specific customer software. Hence we need to include the Embodied Emissions (M) value in the monitoring metric . 
+
+
+3) Application Environment Types
+The usage of the above SRE metric also changes by environment. This statement is tied to the Application architecture factor somewhat but broadly, the concept is that for the purpose of carbon tracking and monitoring, measurement should be done for all environments like development, testing, QA, Performance and Production. This is because the carbon emission of the software increases manifold for lower environments like development and QA for workloads like machine learning models. 
+
+Multiple iterations of running AI experiments in lower environments should be tracked for carbon emissions and hence the scope of the metric should be monitored at the environment level. 
+
+Similarly for the other common workload scenarios like web or desktop applications, multiple performance tests are executed to achieve the SLO targets for throughput and /or latency. Through the process of trying to achieve these targets, the compute and storage resources are used more intensively than it would be on a production environment. Hence tracking of the metric is recommended at Environment scope as well.
+
+**<to be added>**
+
+
+**Conclusion**
+Some of the deductions we have made at the end of this article:
+
+1) Total carbon emissions (C) is the metric we will monitor at an SRE level .
+2) The application of this metric can be done at multiple scope levels - Environment , Hosting infrastructure type and Application architecture
+3) In future iterations of the specification, work to be done to understand the SRE impact of the C metric for other SLO attributes like latency, availability. A brief write up below:
```

Comment:
I don't think the appendix should make comments on what future iterations of the spec will do because we will have to come back and update / link to that progress and that might be difficult to do @srini1978. I would recommend removing this sentence.